### DISCUSSION TITLE
2023년 05월 11일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#52] [General]
# 2023년 05월 11일

LOG 미션을 완료했다! 사실 전날도 matzip 약속 때문에 저녁에 아무것도 못한 게 마음에 걸렸는데 다행이었다. 말랑의 코드를 아침에 보면서 왔는데 많은 도움이 된 것 같다. 말랑 그는 나의 거인. 거인의 어깨 위에서 많은 걸 배웠다. 서비스에 …

https://vero.wiki/2023-05-11/

**Comment by kevstevie** [comment]
>Integer 같은 wrapper 클래스가 아닌 primitive 타입을 사용하고 @Positive 같은 validation 어노테이션을 사용하면 적용되지 않는다.

진짜로요??

**Reply by Cyma-s** [reply]
원래 요청이 String으로 들어와서 Request 변수로 변환되는거라 primitive 타입을 쓰면 적용이 안 됩니다
